packages/components/src/spinner/index.native.js

```diff
@@ -9,7 +9,7 @@ import { View } from 'react-native';
 import style from './style.scss';
 
 export default function Spinner( props ) {
-	const { progress } = props;
+	const { progress = 0 } = props;
```

Comment:
While I was testing on Android I found that using `<Spinner />` made the app to crash, after some investigations I found out that the cause was that `progress` was `undefined`. Surprisingly on iOS was working ok 🤷‍♂️